### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by heidydallard**

### CODE REVIEW

File: include/gepetto/gui/osgwidget.hh [hunk 1]
```diff
@@ -58,7 +58,7 @@ namespace gepetto {
         WindowsManager::WindowID windowID () const;
 
 signals:
-        void selected (QString name, QVector3D positionInWorldFrame);
+        void clicked (QString name, QVector3D positionInWorldFrame);
```
Comment (jmirabel) [inline include/gepetto/gui/osgwidget.hh]: Changing this name means the documentation of the plug-ins has to be updated.

File: include/gepetto/gui/selection-handler.hh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    private slots:
+      void getBodies(QStringList selectedBodies);
+      void changeMode();
+
+    private:
+      void initWidget();
+
+      std::vector<SelectionMode *> modes_;
+      QComboBox* modesList_;
```
Comment (jmirabel) [inline include/gepetto/gui/selection-handler.hh]: This is not used, isn't it ?

Comment (heidydallard) [inline include/gepetto/gui/selection-handler.hh]: The class or the slots ?

Comment (jmirabel) [inline include/gepetto/gui/selection-handler.hh]: The attribute `modesList_`.

Comment (heidydallard) [inline include/gepetto/gui/selection-handler.hh]: Yes I forgot to remove it.

File: include/gepetto/gui/osgwidget.hh (hunk 1 above)
Comment (heidydallard) [inline include/gepetto/gui/osgwidget.hh]: I didn't fin any reference of this function in the documentation.
If you were thinking about the code, I changed all the code who was using selected.

Comment (jmirabel) [inline include/gepetto/gui/osgwidget.hh]: Python plugins are using this, like `pyplugins/gepetto/gui/pythonwidget.py`.

Comment (heidydallard) [inline include/gepetto/gui/osgwidget.hh]: It was the only one I didn't changed.

**Comment by jmirabel** [comment]
Partially solve #29

**Pull request merged**